### PR TITLE
fixes creativeTab stuff, makes elements register and adds a tank to use in GUIs

### PR DESCRIPTION
**Pull request by way2muchnoise**

### CODE REVIEW

**Comment by jakimfett** [comment]
I haven't gotten this merged yet, but something to keep in mind for future PRs...it's much nicer to have logically grouped changes (eg, all the creative tab stuff, or all the element registration stuff) all in a single commit. I'd rather see five commits with comments explaining what and why than a single commit with less detail that doesn't mention parts of it.

**Comment by way2muchnoise** [comment]
@jakimfett It's fine I'll try and clean it up today

File: src/main/java/minechem/apparatus/tier1/opticalMicroscope/OpticalMicroscopeContainer.java [hunk 1]
```diff
@@ -7,10 +7,10 @@
 public class OpticalMicroscopeContainer extends BasicContainer
 {
     /**
-     * Container object for the workbench
+     * Container object for the opticalMicroscope
      *
      * @param inventoryPlayer   the player's inventory
-     * @param opticalMicroscope the bench TileEntity
+     * @param opticalMicroscope the microscope TileEntity
```
Comment (jakimfett) [inline src/main/java/minechem/apparatus/tier1/opticalMicroscope/OpticalMicroscopeContainer.java]: Nice catch, thanks...I copypasta'd this from my other project.

Comment (way2muchnoise) [inline src/main/java/minechem/apparatus/tier1/opticalMicroscope/OpticalMicroscopeContainer.java]: @jakimfett ikr

**Pull request merged**